native/storybook/rtl-addon/register.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import addons from '@storybook/addons';
+
+const styles = {
```

Comment:
I think this can probably be removed, it doesn't look like it's being used (it's boilerplate from the storybook example addon).